modules/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// * virt/node_maintenance/virt-using-skip-node.adoc
+//
+
+[id="virt-using-skip-node_{context}"]
+
+.Procedure
+
+* Substitute the value of `<node_name>` with the applicable node.
```

Comment:
I might rewrite this as something like `* Annotate the node that you want to skip by running the following command:` and then use a callout to emphasize that `<node_name>` must be replaced